src/resources/postcss/base/forms/_text.pcss

```diff
@@ -24,27 +24,44 @@
 	}
 
 	.tribe-common-form-control-text__input {
-		@mixin desktop-body-2-bold;
-
-		border: 1px solid var(--color-border-default);
-		padding: 12px 18px;
+		@mixin desktop-body-1;
+		border: none;
+		border-bottom: 1px solid var(--color-border-default);
+		padding: 12px 20px 12px 0;
 		width: 100%;
 
+		@media (--viewport-medium) {
+			@mixin desktop-body-2;
+
+			border: none;
+			padding: 12px 18px;
```

Comment:
let's use `--spacer` for these. also, i think these are different now, confirm with the desktop designs.